witness/golang/cmd/witness/example.conf

```diff
@@ -2,7 +2,7 @@
     "logs": [
         {
             "hashStrategy": "default",
-            "logID": "testlog",
+            "logID": "monkeys",
```

Comment:
nit: this is fun, but may be a head scratcher for serious people looking at this example config :-) `examplelog` or `mylogid` would be the safe paths.